src/test/regress/expected/mixed_relkind_tests.out

```diff
@@ -667,6 +667,16 @@ SELECT * FROM unlogged_distributed_table WHERE a = 1 ORDER BY 1,2 FOR UPDATE;
 (1 row)
 
 VACUUM partitioned_distributed_table;
+DEBUG:  not propagating VACUUM command to worker nodes
+HINT:  Set citus.enable_ddl_propagation to true in order to send targeted VACUUM commands to worker nodes.
+DETAIL:  from localhost:xxxxx
+DEBUG:  not propagating VACUUM command to worker nodes
+HINT:  Set citus.enable_ddl_propagation to true in order to send targeted VACUUM commands to worker nodes.
+DETAIL:  from localhost:xxxxx
+DEBUG:  there is no distributed table to propagate to workers in command VACUUM
+DETAIL:  from localhost:xxxxx
+DEBUG:  there is no distributed table to propagate to workers in command VACUUM
+DETAIL:  from localhost:xxxxx
```

Comment:
These messages are very strange, it looks like the command is trying to propagate again from the worker. We should prevent that.